ports/vc/portfile.cmake

```
@@ -1,21 +1,15 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO  VcDevel/Vc
-    REF 1.4.2
-    SHA512 9a929cd48bdf6b8e94765bd649e7ec42b10dea28e36eff288223d72cffa5f4fc8693e942aa3f780b42d8a0c1824fcabff22ec0622aa8ea5232c9123858b8bbbf
+    REF 1.4.3
+    SHA512 7c0c4ccf8c7c4585334482135f2daf1a5bc088114b880093893583bdcea1fbfcec02485da6059304c510c8b1bb1b768ef04fd7ac8ccb21b9ebbad5d0d5babaef
     HEAD_REF 1.4
-    PATCHES
-        correct_cmake_config_path.patch
-        dont-use-uninit-var.patch
+    PATCHES correct_cmake_config_path.patch
 )
 
-vcpkg_configure_cmake(
-    SOURCE_PATH ${SOURCE_PATH}
-    PREFER_NINJA
-    OPTIONS -DBUILD_TESTING=OFF -DBUILD_EXAMPLES=OFF
-)
-vcpkg_install_cmake()
-vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/Vc/)
+vcpkg_cmake_configure(SOURCE_PATH ${SOURCE_PATH})
```

```suggestion
vcpkg_cmake_configure(
    SOURCE_PATH "${SOURCE_PATH}"
    OPTIONS
        -DBUILD_TESTING=OFF
        -DBUILD_EXAMPLES=OFF
    )
```
By default, we disable building unnecessary TEST and EXAMPLES options to reduce build time

---

Makes perfect sense. Examples were off by default in Vc for some time now and the latest release also turned the tests off by default. Do you still want to explicitely set these flags?

---

We can't confirm when upstream will modify these defaults (unless explicitly stated), so I think it's best to add this option.

---

Ok! Fixed.